README.md

```diff
@@ -1,26 +1,63 @@
-# streetscapes
-This repository contains information and code for retrieving and using data from the [global-streetscapes](https://github.com/ualsg/global-streetscapes/tree/main) dataset. 
+# `streetscapes`
+This repository contains information and code for retrieving and using data from the [global-streetscapes](https://github.com/ualsg/global-streetscapes/tree/main) dataset.
 
-### Converting global-streetscapes to parquet
+# Installation
+Clone the repository, enter the root directory and use pip to install the `streetscapes` package:
 
-`convert_csv_to_parquet.py` contains the code for reproducing `streetscapes-data.parquet` dataset. 
-For input you will have to download the original csv files from [hugging face](https://huggingface.co/datasets/NUS-UAL/global-streetscapes/tree/main/data). The code looks for these csv files in `../streetscapes-data`, which you may need to create yourself or change the path to the location of the files. 
-To limit size, the dataset contains the following data:
+```shell
+$> git clone git@github.com:Urban-M4/streetscapes.git
+<...>
+$> cd streetscapes
+$> pip install -e .
+```
 
-    - `contextual.csv`
-    - `metadata_common_attributes.csv`
-    - `segmentation.csv`
-    - `simplemaps.csv`
+### Dependencies
+There are a lot more dependencies in `pyproject.toml` than strictly necessary to run the examples in this repository. They are necessary for running (at least part of) the code in the original `global-streetscapes` repository, specifically the training pipeline (WIP).
 
-It is possible to combine more CSV files if needed. 
+Streetscapes uses a [custom version](https://github.com/Urban-M4/mapillary-python-sdk) of the [Mapillary Python SDK](https://github.com/mapillary/mapillary-python-sdk) which fixes some dependency issues.
 
-### Analysing the data
+## Environment variables
+To facilitate usage across different setups, some environment variables can can be added to an `.env` file (`<repo-root>` is the root directory of the `streetscapes` repository):
 
-`plot_city.ipynb` shows a simple of example of subsetting the dataset and plotting the data. 
-`subset_data.ipynb` shows an example of subsetting the data for image download, similar to [this example](https://github.com/ualsg/global-streetscapes/blob/main/code/download_imgs/sample_subset_download.ipynb)
+- `MAPILLARY_CLIENT_ID`: A Mapillary client ID string (can be obtained via your [Mapillary account](https://www.mapillary.com/app/)).
+- `MAPILLARY_TOKEN`: A Mapillary token string used for authentication when querying Mapillary via their API.
+- `STREETSCAPES_DATA_PATH`: A directory containing data from the `global-streetscapes` projects, such as CSV files (cf. below). Defaults to `<repo-root>/local/streetscapes-data`.
+- `STREETSCAPES_OUTPUT_DIR`: A directory for output files. Defaults to `<repo-root>/local/output`.
 
-### Acknowledgements/Citation
+## CLI
```

Comment:
I had a few problems running the CLI initially, I think it might be nice to also leave the option of just running the code, especially as there is currently only one command (although it nice that there is space to add more now).